test/e2e/delete_test.go

```diff
@@ -250,6 +244,52 @@ func (tc *testContext) testWindowsNodeDeletion(t *testing.T) {
 	require.NoError(t, err, "could not delete test namespace")
 }
 
+// deployWorkloadsToDrain creates workloads that require special options to drain
+func (tc *testContext) deployWorkloadsToDrain() error {
+	// Deploy a DaemonSet and wait until its pods have been made ready on each Windows node. DaemonSet pods cannot be
+	// drained from a Node.
+	// Doing this ensures that WMCO is able to handle containers being present on the instance when deconfiguring.
+	ds, err := tc.deployNOOPDaemonSet()
+	if err != nil {
+		fmt.Errorf("error creating DaemonSet")
```

Comment:
please wrap the error so we have the full error:
`fmt.Errorf("error creating DaemonSet: %w", err)`